task-5/pkg/index/index.go

```diff
@@ -0,0 +1,204 @@
+package index
+
+import (
+	"bufio"
+	"fmt"
+	"go-gavrila/task-5/pkg/crawler"
+	"io"
+	"os"
+	"strconv"
+	"strings"
+)
+
+const content_path = `./content.csv`
+const index_path = `./index.csv`
+const separator = `;`
+
+var index = make(map[string][]int)
+var content = []crawler.Document{}
+
+func LoadFiles() {
+	load_content(load_file(content_path))
+	load_index(load_file(index_path))
+}
+
+// Loads data from file and converts it to strings slice
+func load_file(name string) []string {
+	fmt.Println("loading", name)
+	file, err := os.Open(name)
+	if err != nil {
+		fmt.Println(err)
+		return []string{}
+	}
+	defer file.Close()
+
+	return read(file)
+
+}
+
+// reads data from provided source
+func read(r io.Reader) []string {
+
+	reader := bufio.NewReader(r)
+	var lines []string
+
+	for {
+		line, err := reader.ReadString('\n')
+		if err != nil {
+			break
+		}
```

Comment:
is there a way to shorten 4 lines of code from above? I've tried "for line, err := reader.ReadString('\n'); err != nil {}"but it doesn't work.